addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1,6 +1,8 @@
 odoo.define('web_editor.wysiwyg', function (require) {
 'use strict';
+const core = require('web.core');
 const Widget = require('web.Widget');
+const Dialog = require('web.Dialog');
```

Comment:
It is use here (https://github.com/odoo-dev/odoo/blob/master-odoo-editor-integration-all/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js#L299) but was generating an error since `_t` and `Dialog` were not defined.